opentelemetry-otlp/src/lib.rs

```diff
@@ -1,8 +1,206 @@
-//! # OpenTelemetry OTLP Exporter
+//! The OTLP Exporter supports exporting trace and metric data in the OTLP format to the OpenTelemetry collector. The OpenTelemetry Collector offers a
```

Comment:
This line looks longer than 80 chars. may need to be wrapped.
```suggestion
//! The OTLP Exporter supports exporting trace and metric data in the 
//! OTLP format to the OpenTelemetry collector. The OpenTelemetry Collector offers a
```

Reply:
ah yeah good catch